.travis/check-user-agent.js

```diff
@@ -0,0 +1,35 @@
+const {DEFAULT_USER_AGENT} = require('../build/constants');
+const httpRequest = require("@apify/http-request");
+const parse = require('csv-parse/lib/sync');
+const UAParser = require("ua-parser-js");
+
+const getMajor = (version) => {
+    return version.split(".")[0];
+};
+
+async function main() {
+    const {body} = await httpRequest({url: "https://omahaproxy.appspot.com/all?csv=1"});
+    const csv = await parse(body, {columns: true});
+    const latestStabel = csv.find(line => line.channel === "stable" && line.os === "linux");
+    const UA = new UAParser(DEFAULT_USER_AGENT);
+    const defaultUserAgentVersion = getMajor(UA.getBrowser().version);
+    const latestStableVersion = getMajor(latestStabel.current_version);
+    if (defaultUserAgentVersion === latestStableVersion) {
+        console.log(`Default User-Agent has a correct Chrome version - ${defaultUserAgentVersion}.`)
+        process.exit(0)
+    } else {
+        console.warn("Default User-Agent has incorrect version of Chrome!");
+        console.warn("This can cause more blocking in head-full mode.");
+        console.warn(`Default User-Agent: ${defaultUserAgentVersion} X Latest Chrome: ${latestStableVersion} `);
+        process.exit(1)
+    }
+
+}
+
+try {
+    main();
+
+} catch (e) {
+    console.log("Could not check User-Agent");
```

Comment:
This will silently succeed the script if there's a problem (such as the API not working) and we'll never know.

Reply:
@mnmkng is there a way that the stage would only show kind of warning. I could not find any solution for the only warning.

Reply:
Couldn't find any either. There's a way to make it fail, but not fail the whole build though. Can't remember the option name though. Just make sure that it applies only to this one script and not to the whole test stage. Or create a new stage just for this.

Reply:
it seems to me that you can not fail the job, but only build under certain environment. -https://docs.travis-ci.com/user/customizing-the-build#Rows-that-are-Allowed-to-Fail